src/test/java/com/axlabs/neo/grantshares/GrantSharesGovTest.java

```diff
@@ -273,7 +275,19 @@ public void succeed_endorsing_with_member() throws Throwable {
         assertThat(p.reject, is(0));
         assertThat(p.abstain, is(0));
 
-        // 6. Test emitted "endorsed" event
+        //6. Test the correct quorum votes are stored
+        int memberCount = gov.callInvokeFunction(GET_MEMBERS_COUNT)
+                .getInvocationResult()
+                .getStack()
+                .get(0)
+                .getInteger()
+                .intValue();
+        ProposalStruct proposal = gov.getProposal(id);
+        int expectedQuorumVotes = (memberCount * proposal.quorum + 99) / 100;
```

Comment:
chore: If you use math like this please add a comment of what you're doing. For someone reading this test it should be immediately clear that this bypasses the modulo and rounding down. These are the code parts that can consume unnecessary time for devs looking at this test in the future.